### PR TITLE
Add netlify config as code

### PR DESCRIPTION
**Pull request by michaelbeaumont**
https://docs.netlify.com/configure-builds/file-based-configuration/

### Checklist
- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [ ] Manually tested
- [ ] Added labels for change area (e.g. `area/nodegroup`), target version (e.g. `version/0.12.0`) and kind (e.g. `kind/improvement`)
- [ ] Make sure the title of the PR is a good description that can go into the release notes

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[build]
+  base = "userdocs/"
```
Comment (martina-if) [inline netlify.toml:2]: Why not just leave `base` empty and `command = make -f Makefile.docs build-pages` ?

Comment (michaelbeaumont) [reply netlify.toml:2]: Netlify tried to be magic and installs `requirements.txt` because it finds it in base. Ill try installing explicitly instead

Comment (martina-if) [reply netlify.toml:2]: ah I see

Comment (michaelbeaumont) [reply netlify.toml]: I just saw it also needs `runtime.txt` in the base dir so I'll leave it as `userdocs`

**Pull request merged**